tahoe_sites/models.py

```diff
@@ -22,7 +22,10 @@ class UserOrganizationMapping(models.Model):
     """
     user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, related_name='+')
     organization = models.ForeignKey(Organization, on_delete=models.CASCADE, related_name='+')
-    is_active = models.BooleanField(default=True)
```

Comment:
Please use a softer deletion until we remove this module:

```py
# TODO: Remove this field after fully migrating to this package
#             see https://github.com/appsembler/tahoe-sites/issues/27
#             the field is kept to help smooth the migration
_deprecated_field_is_active = models.BooleanField(default=True, db_column='is_active')
```

Reply:
Makes sense, thanks @OmarIthawi . Done